lib/utils.sh

```diff
@@ -10,7 +10,7 @@ fi
 
 ASDF_NODEJS_KEYRING=asdf-nodejs.gpg
 
-ASDF_NODEJS_PLUGIN_NAME="$(basename "$(dirname "$(dirname "${BASH_SOURCE[0]}")")")"
+ASDF_NODEJS_PLUGIN_NAME="$(basename "$(dirname "$(dirname $0)")")"
```

Comment:
You need to quote `$0` here, because the path to the script may contain spaces

```suggestion
ASDF_NODEJS_PLUGIN_NAME="$(basename "$(dirname "$(dirname "$0")")")"
```